File: backend/app/internal/core/entities/product.go
package entities

type ProductID [12]byte

type Product struct {
	ID          ProductID  `json:"id" bson:"_id,omitempty"`
	Name        string     `json:"name" bson:"name"`
	Description string     `json:"description" bson:"description"`
	CategoryID  CategoryID `json:"categoryId" bson:"category_id"`
	Amount      int64      `json:"amount" bson:"amount"`
}

File: web/src/api/models/Product.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ImageList } from './ImageList';

export type Product = {
    id: number;
    categoryId: number;
    name: string;
    description?: string;
    amount: number;
    price: number;
    images: ImageList;
};

